### ISSUE TITLE
add a contains sentence for a11y rgaa

### ISSUE THREAD

**Issue by luifr10** [Orange-OpenSource/uuv#424]
Add sentence to check that RGAA a11y result summary contains some result : `I should have the following partial result based on the rgaa reference`
